crate/pages/blog/crate-release-v0-46-4.html

```diff
@@ -0,0 +1,103 @@
+title: New Crate release: v0.46.4
+link: https://crate.io/blog/crate release-v0-46-4/
+author: Kenan
+description: New Crate Release v0.46.4
+created: 2015-02-06
+post_name: new release: v0.46.4
+status: publish
+post_type: post
+tags: crate, release, schemata, column policy
+category: developernews
+
+We're pleased to announce that Crate 0.46.x is now stable. You can find a summary of changes in the [CHANGES.txt](https://github.com/crate/crate/blob/master/CHANGES.txt) file in our GitHub repository. If you find this to not be the case, please do not hesitate to [contact us](https://crate.io/about/contact/) and let us know.
```

Comment:
what I meant with 0.46.x is that we need to take care which version will become stable (probably 0.46.5)